### DISCUSSION TITLE
zsh alias version

### DISCUSSION THREAD

**Discussion by ghost** [CleanMachine1/DollarSkip#48] [General]
This entire program can be put into a single zsh alias, which will make it much smaller and faster (the system will not have to seek the binary from disk). It also resolves issues like #46 because everything is parsed by the shell.
```zsh
alias \$=""
```
We can also use this to add features such as running commands as root (#38)
```zsh
alias #="sudo"
```
This exact setup won't in bash, however I'm sure something similar is possible.

**Comment by CleanMachine1** [comment]
Hi, if this is a better version for you feel free to just use it, Dollarskip is really inefficient.

If you would like to add this feature via a PR in some way, go ahead!
